libafl/src/events/mod.rs

```diff
@@ -451,108 +442,140 @@ pub trait EventFirer: UsesState {
     fn should_send(&self) -> bool;
 }
 
-/// [`ProgressReporter`] report progress to the broker.
-pub trait ProgressReporter: EventFirer
+/// Default implementation of [`ProgressReporter::maybe_report_progress`] for implementors with the
+/// given constraints
+pub fn default_maybe_report_progress<PR, S>(
```

Comment:
`std` instead of `default`? That's our naming scheme in the repo so far (because Default is heavily abused in Rust already)

Reply:
:+1: Makes sense